gta/map.py

```diff
@@ -14,106 +14,229 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+# Initially ported from OpenRW
+# https://github.com/rwengine/openrw/blob/master/rwengine/src/loaders/LoaderIPL.cpp
+
+from enum import Enum
+from collections import namedtuple
+
+# Data types
+Section_INST = namedtuple("Section_INST"  , "id model posX posY posZ scaleX "+
```

Comment:
This looks ugly, I tried one parameter per line but that might be even worse..

Reply:
You can actually also do this

```python
Section_ZONE = namedtuple(
    "Section_ZONE",
    [
        "name", "type", "minX", "minY",
        "minZ", "maxX", "maxY", "maxZ",
        "island", "gangCarDensityDay", "gangCarDensityNight", "gangDensityDay",
        "gangDensityNight", "pedGroupDay", "pedGroupNight",
    ]
)
```